wee_slack.py

```diff
@@ -3488,6 +3635,34 @@ def command_users(data, current_buffer, args):
     return w.WEECHAT_RC_OK_EAT
 
 
+@slack_buffer_required
+@utf8_decode
+def command_usergroups(data, current_buffer, args):
+    """
+    /slack usergroups
+    List the usergroups in the current team
+    / slack usergroups <handler>
+    List the users in a usergroup using handler
+    """
+    e = EVENTROUTER
+    team = e.weechat_controller.buffers[current_buffer].team
+    if args and args in team.subteam_handles_to_id.keys():
+        subteam_identifer = team.subteam_handles_to_id[args]
+        subteam = team.subteams[subteam_identifer]
+        s = SlackRequest(team.token, "usergroups.users.list", { "usergroup": subteam.identifier }, team_hash=team.team_hash)
+        e.receive(s)
+    elif not args:
+        team.buffer_prnt("Usergroups:")
+        for subteam in team.subteams.values():
+            team.buffer_prnt("    {:<25}(@{})".format(subteam.name, subteam.handle))
+    else:
+        w.prnt('', 'ERROR: Unknown usergroup handler: {}'.format(args))
+        w.prnt('', 'Usage: /slack usergroups <handler>')
```

Comment:
Don't print usage info when the command syntax was correct. The first message is sufficient.